src/components/Keyword/ModifyKeyword/index.js

```diff
@@ -0,0 +1,114 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import KeywordHeader from '../KeywordHeader';
+import { getRecommendation } from 'store/modifyKeyword';
+import * as S from './styles';
+import { useDispatch, useSelector } from 'react-redux';
+import KeywordAlarm from '../KeywordAlarm';
+import { AlarmContext } from 'context/KeywordAlarmContext';
+import { useContext } from 'react';
+
+const ModifyKeyword = () => {
+  const [site, setSite] = useState('');
+  const [recommendList, setRecommendList] = useState([]);
+  const [selectRecommendItem, setSelectRecommendItem] = useState([]);
+  const [alreadyRegisterItem, setAlreadyRegisterItem] = useState(false);
```

Comment:
```suggestion
  const [isAlreadyRegisterItem, setIsAlreadyRegisterItem] = useState(false);
```
위에 식으로 고치는 것이 좋아 보입니다

Reply:
변수이름이 조금 뭔지 모르겠군요 이미회원가입아이템? 잘 해석이 되지 않습니다,

Reply:
이미 등록된 아이템 입니다!